filtres/nou.py

```diff
@@ -44,6 +44,16 @@ def netejar_html(self, html):
         else:
             return html
 
+    def cal_enviar_cc(self):
+        if len(self.msg.get_cc()) == 0:
+            return False
+        if settings.get("afegir_solicitats_addicionals_en_cc"):
+            return True
+        patro = settings.get("afegir_solicitats_addicionals_en_cc_nomes_via")
+        if not patro:
+            return False
+        return (re.compile(patro).match(self.msg.get_to()))
```

Comment:
He hagut de mirar el codi del `get_to()` per assegurar-me que té en compte la capçalera `Resent-To` (efectivament la té en compte).

No correspondria fer el canvi aquí però potser caldria considerar un canvi de nom i dir-ne `get_recipient` (per exemple) per evitar l'ambigüitat.